pkgs/development/interpreters/quickjs/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "quickjs-${version}";
+  version = "2019-07-09";
+
+  src = fetchurl {
+    url = "https://bellard.org/quickjs/quickjs-${version}.tar.xz";
+    sha256 = "0r6mgh0m85c11y8ah5iphzpw518br0hi36f92mgdg2iivpciq31m";
+  };
+
+  configurePhase = ''
+    sed -e "s/^CONFIG_M32=y/#&/" -i Makefile
+  '';
+
+  buildPhase = ''
```

Comment:
Both `buildPhase` and `installPhase` can be replaced with `makeFlags = [ "prefix=$(out)" ];`